src/services/apimPolicyBuilder.ts

```diff
@@ -142,13 +149,47 @@ export class ApimPolicyBuilder {
       "required-claims": claims
     };
 
-    Object.keys(elements).forEach((key) => {
-      if (!elements[key]) {
-        delete elements[key];
+    Object.keys(policy).forEach((key) => {
+      if (!policy[key]) {
+        delete policy[key];
+      }
+    });
+
+    this.policyRoot.policies.inbound[0]["validate-jwt"] = [policy];
```

Comment:
is this referenced multiple places? if so, promote to a const.

Reply:
Only in this 1 place :)